.buildkite/build-ios.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash -eu
+
+echo '--- :node: Setup Node depenendencies'
+echo '--- :node: 1. Install nvm'
+brew install nvm
+
+echo '--- :node: 2. Load nvm in the current shell'
+export NVM_DIR="$HOME/.nvm"
+mkdir -p "$NVM_DIR"
+[ -s "$HOMEBREW_PREFIX/opt/nvm/nvm.sh" ] && \. "$HOMEBREW_PREFIX/opt/nvm/nvm.sh" --install
+
+echo '--- :node: 3. Install node version from .nvmrc'
+nvm install "$(cat .nvmrc)" && nvm use
```

Comment:
An alternative to the manual installation above is using [nvm-buildkite-plugin](https://github.com/Automattic/nvm-buildkite-plugin). You can simply add the plugin `automattic/nvm#0.1.0` to the pipeline.yml file and remove the lines above.

Reply:
@crazytonyli I tried to use the new nvm plugin, but run into a setup issue https://github.com/Automattic/nvm-buildkite-plugin/issue/6.

In my previous mention of that issue, I supposed I could have misunderstood the target platform. From your comment, maybe I did something wrong in the setup.

Reply:
Given my current understanding of there being an issue in adopting the plugin, I decided to enable auto-merge. If it turns out I was wrong and it's just a matter of me not having configured it properly, then I'll happily followup with a PR to remove the manual Node setup

Reply:
Thanks for reporting the issues. I can have a look at them later. 👍 

> If it turns out I was wrong and it's just a matter of me not having configured it properly

No. Those issues should definitely be solved on the plugin side.